acq_stat_reports/config.py

```diff
@@ -0,0 +1,21 @@
+class Config:
+    # "Top-level data directory",
+    data_dir = "."
+
+    # "Default figure width in inches",
+    figure_width = 5
+
+    # "Default figure height in inches",
+    figure_height = 2.5
+
+    # "Close matplotlib figures after plotting",
+    close_figures = False
```

Comment:
Remember that this PR is about getting new plots. I wanted to be done with this more or less quickly so I could improve the plots themselves and perhaps use plotly, at which point this will not matter.

Still, I added it to PR #9